joblib/test/test_numpy_pickle.py

```diff
@@ -371,6 +542,215 @@ def test_joblib_pickle_across_python_versions():
     for fname in data_filenames:
         _check_pickle(fname, expected_list)
 
+
+def _check_compression_format(filename, expected_list):
+    if (sys.version_info[:2] < (3, 3) and (filename[-2:] == 'xz' or
+                                           filename[-4:] == 'lzma')):
+        # lzma is not supported for python versions < 3.3
+        nose.tools.assert_raises(NotImplementedError,
+                                 numpy_pickle.load, filename)
+    else:
+        _check_pickle(filename, expected_list)
+
+
+@with_numpy
+def test_joblib_decompression_format_support():
+    # XXX: temporarily disable this test on non little-endian machines
```

Comment:
IIRC this PR fixed this problem so you can remove this SkipTest. There was an issue about this I think.